File: platform/sdk/nodejs/examples/cloud/cloud-026_lifecycle_warm_to_cold_180d.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("life-bucket");
const result = client.lifecycleWarmToCold180d("life-bucket");
if (!result.ok || result.rule.transition.days !== 180) process.exit(1);
console.log("XJ-OK: cloud-026_lifecycle_warm_to_cold_180d");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-026_lifecycle_warm_to_cold_180d.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("life-bucket")
result = client.lifecycleWarmToCold180d("life-bucket")
if not result["ok"] or result["rule"]["transition"]["days"] != 180:
    sys.exit(1)
print("XJ-OK: cloud-026_lifecycle_warm_to_cold_180d")
sys.exit(0)
